domain/lib/src/extension/string_extension.dart

```diff
@@ -33,4 +33,10 @@ import 'package:domain/domain.dart';
 extension StringExtension on String {
   int compareToSort(String value, OrderType orderType) =>
       toLowerCase().compareTo(value.toLowerCase()) * (orderType == OrderType.ascending ? -1 : 1);
+
+  String toMiddleEllipsis()
+  {
+    final subLength = length - split('.').last.length;
+    return '...' + substring(subLength-3,subLength) + split('.').last;
+  }
```

Comment:
it is not C#. 
```dart
String toMiddleEllipsis() {
    final subLength = length - split('.').last.length;
    return '...' + substring(subLength-3,subLength) + split('.').last;
}
```
Please try to follow the convention in code base.

Reply:
I fixed it and pushed it up. but still use -f. when not using -f has reported reject error. Hope you help me. Thank you, please